wisp-task/src/main/kotlin/wisp/task/RepeatedTaskMetrics.kt

```diff
@@ -14,4 +18,15 @@ class RepeatedTaskMetrics(meterRegistry: MeterRegistry) {
       .publishPercentileHistogram()
       .register(meterRegistry)
 
+  internal var successCount: Counter = Counter.builder("repeated_task_success_count")
+    .description("count of successful repeated tasks")
+    .register(meterRegistry)
+
+  internal var retryCount: Counter = Counter.builder("repeated_task_retry_count")
+    .description("count of repeated tasks retries")
+    .register(meterRegistry)
+
+  internal var noWorkCount: Counter = Counter.builder("repeated_task_no_work_count")
+    .description("count of repeated tasks invocations with no work to do")
+    .register(meterRegistry)
```

Comment:
I completely missed that we're starting to use Micrometer for Misk. If so, the naming is off and should be e.g. `repeated.task.success`, with the naming convention for Prometheus kicking in: https://micrometer.io/docs/concepts#_naming_meters

Unless of course these are existing metric names, in which case `repeated_task_duration` doesn't follow Prometheus convention, and we should probably either drop Micrometer or add a naming convention class.

Reply:
I'll switch to '.' convention for micrometer to not assume what's behind it.

It's a new implementation for the same idea behind the misk version, so there's some extensions to what was already there (just the duration).